stdlib/LLVMLibUnwind_jll/src/LLVMLibUnwind_jll.jl

```diff
@@ -14,18 +14,18 @@ export llvmlibunwind
 # These get calculated in __init__()
 const PATH = Ref("")
 const LIBPATH = Ref("")
-artifact_dir = ""
+const artifact_dir = ""
 llvmlibunwind_handle = C_NULL
-llvmlibunwind_path = ""
+const llvmlibunwind_path = ""
```

Comment:
This is not correct, `artifact_dir` and `llvmlibunwind_path` are not constant, they're modified inside `__init__`.  If anything, you could type-annotate these global non-constant variables
```julia
artifact_dir::String = ""
llvmlibunwind_handle::Ptr{Cvoid} = C_NULL
llvmlibunwind_path::String = ""
```

Reply:
Okay, there we go